packages/expo-modules-core/ios/Core/Views/ViewModuleWrapper.swift

```diff
@@ -20,7 +20,7 @@ public final class ViewModuleWrapper: RCTViewManager, DynamicModuleWrapperProtoc
    A reference to the module holder that stores the module definition.
    Enforced unwrapping is required since it can be set right after the object is initialized.
```

Comment:
```suggestion
```

also the variable called `wrappedModuleHolder` doesn't sound precise afterall. maybe we could refactor as `moduleHolder` as now it can be nil.

Reply:
Yes, renaming it makes sense. In practice, it was never nil before so it should never be nil now. We're just changing how it's stored so arc won't increase the reference count.